### ISSUE TITLE
Good now every time results in identification of an extension causing an issue

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#111672]
Testing #111495

I clicked Good now on every prompt shown and still saw at the end "Extension Bisect is done and has identified youssef.viow as the extension causing the problem".  Right now, it seems the code is assuming that an extension is to blame for the behavior, when that might not be the case. Perhaps it should check as the first pass with all extensions disabled if the problem persists and only if that is true, continue to bisect?

**Comment by jrieken** [comment]
> Perhaps it should check as the first pass with all extensions disabled if the problem persists and only if that is true, continue to bisect?

I like that

**Comment by jrieken** [comment]
> I clicked Good now on every prompt shown and still saw at the end 

Tho, just to clarify: When you "randomly" say "good now" on all prompts then you do get a false result and there is nothing we can to prevent this

**Comment by jrieken** [comment]
Pushed a change that makes extension bisect start with all extensions disabled. If it's then bad it goes the "might be an issue with vscode"-way. But when always saying "good now" you can get a false result

**Comment by meganrogge** [comment]
@jrieken not seeing this happen yet, thinking it's unreleased at this time

**Comment by jrieken** [comment]
\closedWith 2388291

**Comment by meganrogge** [comment]
still not working for me @jrieken

**Comment by roblourens** [comment]
@meganrogge please reopen these if they still repro

**Comment by jrieken** [comment]
@meganrogge unsure what exactly you have tried but when you always say "Good Now" you won't ever to able to find a bad extension. So, get can be more specific about what's not working?

**Comment by meganrogge** [comment]
@jrieken 

![recording (70)](https://user-images.githubusercontent.com/29464607/101716515-45572380-3a52-11eb-8a12-d86e274c3656.gif)

**Comment by jrieken** [comment]
It is expected that pressing "Good Now" without consideration yields to blaming an unrelated extension. There is nothing to prevent this and adding an extra step where no extension is disabled doesn't make sense. What would we do? Disable no extension, ask the user, and upon replying "good now", we tell them that everything is OK? Then don't start bisect in the first place

**Comment by mjbvz** [comment]
I confirmed the behavior @meganrogge described and agree it's confusing that we tell the user that an extension is to blame:

<img width="444" alt="Screen Shot 2021-01-28 at 7 50 06 PM" src="https://user-images.githubusercontent.com/12821956/106229345-3f540800-61a2-11eb-80d3-9c82d06b45be.png">

@jrieken I know this specific issue is very unlikely to be hit, but shouldn't we show a message [like this one](https://github.com/microsoft/vscode/blob/48dbb328998393097c6dc8b0fea4c5bef62bd437/src/vs/workbench/services/extensionManagement/browser/extensionBisect.ts#L295) instead in this case?

**Comment by jrieken** [comment]
You simply cannot do that. Bisect always disables halves of extensions and when you always say "good now" it means the disabled extensions contain the bad extension, this goes on until on single extension is left and that's the bad one.

**Comment by joaomoreno** [comment]
Yeah this is a weird one. Here's what git does:

```
➜  vscode-vsce git:(master) git bisect start
➜  vscode-vsce git:(master) git bisect bad master
➜  vscode-vsce git:(master) git bisect good v1.25.0
Bisecting: 233 revisions left to test after this (roughly 8 steps)
[286a758ebdce1cb1cd2289ab7cbc2e43c89b1d2b] update badges
➜  vscode-vsce git:(286a758) git bisect good
Bisecting: 117 revisions left to test after this (roughly 7 steps)
[2a02458bad775ec2f6d689ebdabb77bf74cc29e5] Merge pull request #411 from felixhao28/master
➜  vscode-vsce git:(2a02458) git bisect good
Bisecting: 58 revisions left to test after this (roughly 6 steps)
[dc49ab513c3357ac1d7dd2af713056c383318ba6] - add extension kind property - add web extension property - include all files as web resources - cap web resources to 25 - add tests
➜  vscode-vsce git:(dc49ab5) git bisect good
Bisecting: 28 revisions left to test after this (roughly 5 steps)
[4b02d71d44faae31841ef607a8872b20bd17db27] Merge pull request #492 from rbuckton/detectYarn
➜  vscode-vsce git:(4b02d71) git bisect good
Bisecting: 14 revisions left to test after this (roughly 4 steps)
[1530b64f573b5d2ecfb53f58f13d54561181b580] fixes #509
➜  vscode-vsce git:(1530b64) git bisect good
Bisecting: 6 revisions left to test after this (roughly 3 steps)
[28044207c84b236d5f8ec965d9046661a3d7add3] Merge pull request #521 from eltociear/patch-1
➜  vscode-vsce git:(2804420) git bisect good
Bisecting: 2 revisions left to test after this (roughly 2 steps)
[7b3b2370ff39003ee17670e60ee9b97404eb70af] Merge pull request #529 from microsoft/aeschli/productIconThemes
➜  vscode-vsce git:(7b3b237) git bisect good
Bisecting: 0 revisions left to test after this (roughly 1 step)
[b910d549f4282502b7dee05365fc38b675323861] Merge branch 'master' of https://github.com/microsoft/vscode-vsce
➜  vscode-vsce git:(b910d54) git bisect good
9f200a5ec31eee2b5e886dd202b73bcd225f1553 is the first bad commit
commit 9f200a5ec31eee2b5e886dd202b73bcd225f1553
Author: João Moreno <joao.moreno@microsoft.com>
Date:   Fri Jan 29 11:11:09 2021 +0100

    fixes #531

 src/package.ts           | 16 ++++++++++++++++
 src/test/package.test.ts | 10 ++++++++++
 2 files changed, 26 insertions(+)
➜  vscode-vsce git:(b910d54)
```

For git, the mere fact that you start bisecting is the acknowledgement that something is wrong. So, when you start bisecting extensions, you clicked on the very first hypothetical `This is bad` button. I think the current behavior is correct.

**Comment by lramos15** [comment]
Marking as verified due to this being intended and it following a similar issue to git bisect. I would say @joaomoreno's new wording makes sense.
